src/Tools/dotnet-user-secrets/test/SecretManagerTests.cs

```diff
@@ -149,19 +149,20 @@ public void SetSecrets(bool fromCurrentDirectory)
         }
 
         [Fact]
+        [QuarantinedTest]
```

Comment:
I was going to suggest throwing these in here ;).

Reply:
You can still suggest it ;)